### DISCUSSION TITLE
How to see the lineage after runnig absaoss/spline-spark-agent

### DISCUSSION THREAD

**Discussion by brave-lee** [AbsaOSS/spline#1184] [Q&A]
I am deployed and can access the ui. How to see the lineage in the example after running the example ?
![image](https://user-images.githubusercontent.com/42576980/221816362-12259e13-c3b3-4a29-96e4-c4c37aefe47a.png)

docker run --rm -e SPLINE_PRODUCER_URL=http://xxx.xxx:8080/producer absaoss/spline-spark-agent:1.0.4
![image](https://user-images.githubusercontent.com/42576980/221816693-b6cb9035-37b4-4b04-bd4a-11ced15d0dd7.png)
![image](https://user-images.githubusercontent.com/42576980/221817035-3ea611f2-930c-47a4-9fc9-f00b31f93e6c.png)

**Comment by wajda** [comment]
On your screenshot is the info page of the server. Spline UI is another app - https://hub.docker.com/r/absaoss/spline-web-ui
If you use our demo Docker compose setup then try the port 9090.

**Reply by brave-lee** [reply]
Yes, you are right. I made a mistake
